httpcore5/src/main/java/org/apache/hc/core5/http/impl/bootstrap/AsyncRequester.java

```diff
@@ -132,4 +119,8 @@ public void close() throws IOException {
         ioReactor.close();
     }
 
+    @Override
```

Comment:
@clevertension Do we need to expose this method?

Reply:
this method must implement in the AsyncRequester, AsyncServer, because the 
AbstractAsyncServerAndRequester should use the IORector to connect, so change it to **protected** or **friendly**?